client/container_copy.go

```diff
@@ -42,7 +42,7 @@ func (cli *Client) CopyToContainer(ctx context.Context, container, path string,
 		query.Set("copyUIDGID", "true")
 	}
 
-	apiPath := fmt.Sprintf("/containers/%s/archive", container)
+	apiPath := "/containers/" + container + "/archive"
```

Comment:
I think for now this is ok, as it's closest to what the existing code does. We should do a whole scan of all URL's we generate, because there's no escaping / handling of special characters in most locations